pyspades/player.py

```diff
@@ -1217,11 +1217,20 @@ def continue_map_transfer(self) -> None:
     def send_data(self, data):
         self.protocol.transport.write(data, self.address)
 
-    def send_chat(self, value: str, global_message: bool = False) -> None:
+    def send_chat(self, value: str, global_message: bool = False, custom_type: int = CHAT_ALL) -> None:
         if self.deaf:
             return
         chat_message = loaders.ChatMessage()
-        if not global_message:
+        if custom_type > 2 and "client" in self.client_info:
+            if self.client_info["client"] == "OpenSpades":
+                value = OPENSPADES_CHATTYPES[custom_type] + value
+            elif EXTENSION_CHATTYPE in self.proto_extensions:
```

Comment:
I think the extension should be checked first because it should be preferred to the old OpenSpades way, but if we're going to send the OpenSpades markup to everyone then that's going to happen either way

Reply:
Yes, i was checking for OS first because though of not sending the "prefix" for everyone, but since its just two characters, why not?